README.md

```diff
@@ -100,6 +100,37 @@ in the transactions which are fetched over JSON-RPC.
 - [ ] FFI Bindings (see note)
 - [ ] CLI for common operations
 
+### Websockets
+
+Websockets support is turned on via the feature-flag `ws`:
+
+```toml
+[dependencies]
+
+ethers = { git = "https://github.com/gakonst/ethers-rs", features = ["ws"] }
+```
+
+### Interprocess Communication (IPC)
+
+IPC support is turned on via the feature-flag `ipc`:
+
+```toml
+[dependencies]
+
+ethers = { git = "https://github.com/gakonst/ethers-rs", features = ["ipc"] }
+```
+
+### HTTP Secure (HTTPS)
+
+If you are looking to connect to a HTTPS endpoint, then you need to enable TLS (`rustls`) and/or SSL (`openssl`) 
+feature-flags.
+
+```toml
+[dependencies]
+
+ethers = { git = "https://github.com/gakonst/ethers-rs", features = ["openssl", "rustls"] }
```

Comment:
I would rather recommend either-or. Enabling both doesn't make much sense.

so let's simply add two toml examples for openssl and rustls